src/components/Form/Step2/Step2.js

```diff
@@ -36,29 +36,27 @@ const Step2 = ({ currentStep, setCurrentStep, isPaperTicket }) => {
       text: 'I pay monthly by Direct Debit',
       value: 'DirectDebit',
     },
-    {
-      text: 'I bought it from the West Midlands Network or Swift website',
-      value: 'SwiftPortal',
-    },
     {
       text: 'I pay for it through my company',
       value: 'Corporate',
     },
     {
       text:
-        'I bought it from a ticket office, West Midlands Network travel shop or Payzone shop',
+        'I bought it from a travel shop or Payzone shop',
       value: 'Shop',
     },
   ];
 
   // If the user has selected something other than paper ticket in step 1
   if (!isPaperTicket) {
+    const SwiftPortal = { text: 'I bought it from the West Midlands Network or Swift website', value: 'SwiftPortal' };
     const workwise = { text: 'I am on the Workwise scheme', value: 'Workwise' };
     const ticketMachine = {
-      text: 'I bought it from a Swift ticket machine',
+      text: 'I bought it from a Swift kiosk',
       value: 'SwiftPortal',
     };
 
+    radios.splice(-1, 0, SwiftPortal); // push SwiftPortal radio option to last before 1 in radio list   
```

Comment:
```suggestion
    radios.splice(-1, 0, swiftPortal); // push SwiftPortal radio option to last before 1 in radio list   
```

Lowercase first letter for JavaScript variables.